### PR TITLE
Feature/plot filecounts atm ocn increments merge

### LINKED ISSUE #24: Add plotting tools for file counts and increments
Plotting tools exist in feature branches as stand alone python scripts but need cleaned up. A couple issues to consider:

- x-axis formatting: labels and tick marks should range from the start date to end date and be independent of the available data. This makes more sense when the streams are in progress but we wish to start viewing time series that account for the entire stream's time period
- Remove unnecessary commented out blocks of code before merging into develop
- Fix SQL filtering so that only targeted data are pulled from the database

### PR DESCRIPTION
**Pull request by chesleymccoll**
This addresses the issue #24 

This PR brings in the plotting tools in order to generate the png files for monitoring.  Both the file_counts and incremental and mean values for selected variables.  Specific modifications are for plot_increments and plot_filecount:

- Added file to output unexpected file counts
- Added extension of time axis. Including new labels and label sizes
- New labels and label sizes, and corrected typo in the Dict4 of stream3 stream
- Fixed legend output to only be unique labels
- verifying the metric type incremental or mean
- code to double check if the metric is ocean or atmosphere

This code is currently working, and is providing the plots for the monitoring on website.

### CODE REVIEW

File: src/plot_file_counts.py [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+                     'experiments': [{'graph_color': 'black',
+                                      'graph_label': 'Number of files',
+                                      'name': 'replay_stream1',
+                                      'wallclock_start': '2023-07-08 16:25:57'}],
+                     'fig_base_fn': 'files',
+                     'stat_groups': [{'cycles': [0, 21600, 43200, 64800],
+                                      'metrics': ['count'],
+                                      'stat_group_frmt_str':
+                                      'file_{metric}'}],
+                     'work_dir': '/contrib/Chesley.Mccoll/replay/results'}
```
Comment (amschne) [inline src/plot_file_counts.py]: maybe we should consider using a common directory or environment variable

Comment (amschne) [reply src/plot_file_counts.py]: changed work_dir to /contrib/shared/replay/results throughout

**Review by amschne — APPROVED** [review]
After changing the work_dir to a shared directory, I am able to reproduce all of the file counts and increments figures. I think this is ready to merge!

**Comment by amschne** [comment]
pinging @jrknezha - this PR (increments and file count plotting scripts) is running successfully and is ready for your review

**Review by jrknezha — COMMENTED** [review]
Reviewed and ran it fine. Recommendation as per our discussion: it would be easier for a user to be able to edit the work_dir as a global variable at the top of the script instead of the multiple changes that needed to be done to update to the shared folder or anywhere a user wants the results

**Comment by amschne** [comment]
> Reviewed and ran it fine. Recommendation as per our discussion: it would be easier for a user to be able to edit the work_dir as a global variable at the top of the script instead of the multiple changes that needed to be done to update to the shared folder or anywhere a user wants the results

added global constant WORK_DIR as suggested

**Review by jrknezha — APPROVED** [review]
Looks good!

**Pull request merged**